influxql/engine_test.go

```diff
@@ -138,6 +138,42 @@ func TestPlanner_Plan_Mean(t *testing.T) {
 	}
 }
 
+// Ensure the planner can plan and execute a min query
+func TestPlanner_Plan_Min(t *testing.T) {
+	tx := NewTx()
+	tx.CreateIteratorsFunc = func(stmt *influxql.SelectStatement) ([]influxql.Iterator, error) {
+		return []influxql.Iterator{
+			NewIterator(nil, []Point{
+				{"2000-01-01T00:00:00Z", float64(100)},
+				{"2000-01-01T00:00:10Z", float64(200)},
+				{"2000-01-01T00:00:20Z", float64(300)},
+				{"2000-01-01T00:00:30Z", float64(400)},
+			}),
+			NewIterator(nil, []Point{
+				{"2000-01-01T00:01:00Z", float64(0)},
```

Comment:
Let's change this data so that the minimum value is neither at the start nor the end, like it is in the other two tests.